fe1-web/store/stores/IndexedDBStore.ts

```diff
@@ -0,0 +1,70 @@
+import { get, set, update } from 'idb-keyval';
```

Comment:
It seems this file is doing nothing but wrapping around idb-keyval, do we still need it at all? 
Why not directly doing the get/set/update in the Wallet file?